### ISSUE TITLE
Update Brodie dataset citation text

### ISSUE THREAD

**Issue by pvangay** [microbiomedata/nmdc-server#374]
Update the dataset citation on [Eoin's landing page](https://data.microbiomedata.org/details/study/gold:Gs0135149) to: 
Sorensen, P., Brodie, E., Beller, H., Wang, S., Bill, M., & Bouskill, N. (2019). Sample Collection Metadata for Soil Cores from the East River Watershed, Colorado collected in 2017 [Data set]. ESS-DIVE. https://doi.org/10.21952/WTR/1573029

@dehays should this edit be done further upstream as well? FYI, this was a request from Joan at ESS DIVE (since this is an ESS DIVE dataset citation)

**Comment by jbeezley** [comment]
We do have that citation in the [supplementary file](https://github.com/microbiomedata/nmdc-server/blob/master/nmdc_server/ingest/study_additional.json#L120].  The primary DOI coming from the data pipeline is the one that is shown `10.25585/1488224`.

The supplementary DOI's are being returned by the API, and I thought at one point they were being displayed.  @subdavis Do you know if this was intentionally changed or is it a bug?

In any case, if we want the primary DOI changed, then that will need to happen upstream.  Otherwise, I think we just need to fix the issue on the web client.

**Comment by dehays** [comment]
@pvangay  Yes, this sort of edit should happen upstream.  That topic of study attributes is related to the discussion in microbiomedata/nmdc-schema#41. I do not believe Jon / search application, should have responsibility to merge from multiple sources of study attributes.

The issue is then how to add/update these attributes.  They are not available from the original study source (GOLD) so the place to add/update would be in the metadata store from which Jon draws metadata for his relational tables behind search.  That'd mean editing in Mongo or TerminusDB - there is not currently any curation UI.
